plugins/woocommerce-admin/client/marketplace/components/content/content.tsx

```diff
@@ -27,6 +31,18 @@ export default function Content(): JSX.Element {
 	// Get the content for this screen
 	useEffect( () => {
 		const abortController = new AbortController();
+
+		// we are recording both the new and legacy events here for now
+		// they're separate methods to make it easier to remove the legacy one later
+		const marketplaceViewProps = {
+			view: selectedTab,
+			searchTerm: query?.term,
+			productType: query?.section,
```

Comment:
Tracks doesn't link camel case – it compares against [this](https://github.com/woocommerce/woocommerce/blob/f05e8efc17683b56d4941adb702baa16fe8e8cf5/plugins/woocommerce/includes/tracks/class-wc-tracks-event.php#L26) regex and throws error `A valid prop name must be specified`.

```suggestion
			search_term: query?.term,
			product_type: query?.section,
```

Reply:
Ah, good point! I'll rework things to fit snake case then.